src/gui/BandSplitter/TriStateButtonAttachment.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+namespace gui::band_splitter
+{
+enum class BandState;
+class TriStateButtonAttachment : public juce::Button::Listener
```

Comment:
So the issue that I'm seeing with this implementation is that it won't respond to automation changes. I think the best way to work this out would be to use a [`chowdsp::ParameterAttachment`](https://github.com/Chowdhury-DSP/chowdsp_utils/blob/master/modules/plugin/chowdsp_plugin_state/Frontend/chowdsp_ParameterAttachment.h) with a custom callback.